docs/user/filesystem.md

```diff
@@ -218,6 +218,265 @@ dfuse           9.4G  326K  9.4G   1% /tmp/dfuse
+Performance domain affinity level of RP 3
+Server Checksumming                     off
+Health                                  HEALTHY
+Access Control List                     A::OWNER@:rwdtTaAo, A:G:GROUP@:rwtT
+
+$ mount.fuse3 dfuse /tmp/daos_dfuse_samir -o pool=samirrav_pool,container=samirrav_cont
+$ touch /tmp/daos_dfuse_samir/foo
+$ ls -l /tmp/daos_dfuse_samir/
+total 0
+-rw-rw-r-- 1 samirrav samirrav 0 Sep 16 21:24 foo
+$ df -h | grep fuse
+dfuse                         537G  5.1G  532G   1% /tmp/daos_dfuse_samir
+$
+```
+
+#### 2> Add dfs entry in /etc/fstab and mount as part of "mount -a" command.
+
+  Only root user can use "mount -a" command to mount the daos container mention in /etc/fstab file
+  so make sure pool has the correct root ownership.
+
+```bash
+$ dmg -o /home/samirrav/scripts/EC_Demo/test_daos_dmg.yaml pool create --scm-size=8G --nvme-size=64G --label=admin_pool
+Creating DAOS pool with manual per-engine storage allocation: 8.0 GB SCM, 64 GB NVMe (12.50% ratio)
+Pool created with 11.11%,88.89% storage tier ratio
+--------------------------------------------------
+  UUID                 : 13d340c1-34bc-4ada-991b-372572e85c57
+  Service Ranks        : [1-5]
+  Storage Ranks        : [0-7]
+  Total Size           : 576 GB
+  Storage tier 0 (SCM) : 64 GB (8.0 GB / rank)
+  Storage tier 1 (NVMe): 512 GB (64 GB / rank)
+
+$ daos cont create admin_pool admin_cont --type=POSIX
+  Container UUID : c9fea19a-064e-4c0c-84e9-0ad752facb9e
+  Container Label: admin_cont
+  Container Type : POSIX
+
+Successfully created container c9fea19a-064e-4c0c-84e9-0ad752facb9e
+$ daos cont get-prop admin_pool admin_cont
+Properties for container admin_cont
+Name                                    Value
+----                                    -----
+Highest Allocated OID                   0
+Checksum                                off
+Checksum Chunk Size                     32 KiB
+Compression                             off
+Deduplication                           off
+Dedupe Threshold                        4.0 KiB
+EC Cell Size                            64 KiB
+Performance domain affinity level of EC 1
+Encryption                              off
+Global Version                          2
+Group                                   root@
+Label                                   admin_cont
+Layout Type                             POSIX (1)
+Layout Version                          1
+Max Snapshot                            0
+Owner                                   root@
+Redundancy Factor                       rf0
+Redundancy Level                        node (2)
+Performance domain affinity level of RP 3
+Server Checksumming                     off
+Health                                  HEALTHY
+Access Control List                     A::OWNER@:rwdtTaAo, A:G:GROUP@:rwtT
+
+$ echo '/usr/bin/dfuse /tmp/root_dfuse fuse3 pool=admin_pool,container=admin_cont,auto,x-systemd.requires=daos_agent.service    0 0' >> /etc/fstab
```

Comment:
Probably best not to mount under /tmp in the examples, it's bad practice and we should avoid promoting it.
```suggestion
$ echo 'dfuse /scratch/daos_root fuse3 pool=admin_pool,container=admin_cont,auto,x-systemd.requires=daos_agent.service    0 0' >> /etc/fstab
```